store/servers.ts

```diff
@@ -56,16 +68,15 @@ export const actions: ActionTree<ServerState, ServerState> = {
               address: serverUrl
             });
           }
-
-          this.$router.push('/login');
+          if (!silent) this.$router.push('/login');
         }
-      } else {
+      } else if (!silent) {
```

Comment:
This one will open a snackBar with that `server version is low` even if it is not.